stable/enterprise/templates/_helpers.tpl

```diff
@@ -154,4 +154,85 @@ secretName: {{ . }}
 {{- else }}
 secretName: {{ template "enterprise.fullname" . }}-license
 {{- end }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{- define "checkDriverEnabled" -}}
```

Comment:
NIT: It would be good to have a comment on these template definitions (or even each piece of logic) explaining what they are doing. Go templating isn't the easiest code to read.